templates/gce.tf.erb

```diff
@@ -1,12 +1,6 @@
-provider "google" {
-  region      = "<%= ENV['GCE_REGION'] %>"
-  project     = "govuk-publishing-dns"
-  credentials = "${file("<%= ENV['GCE_CREDENTIALS'] %>")}"
-}
-
 <% records['records'].each_with_index do | record, index | %>
 resource "google_dns_record_set" "<%= record['subdomain'].gsub('.', '_').gsub('@', 'AT') %>_<%= index %>" {
```

Comment:
Now that I think about it this could be awkward in terms of e.g. inserting a new record at the top and then redoing all the other records.